src/lib/menu/menu-trigger.ts

```diff
@@ -85,7 +85,7 @@ export class MatMenuTrigger implements AfterContentInit, OnDestroy {
   private _portal: TemplatePortal;
   private _overlayRef: OverlayRef | null = null;
   private _menuOpen: boolean = false;
-  private _closeSubscription = Subscription.EMPTY;
+  private _closingActionsSubscription = Subscription.EMPTY;
```

Comment:
Would it make sense to refactor this to use the  `takeUntil(destroyed)` pattern now? Looks like there's a few manually managed subscriptions.

Reply:
Actually, I've had to revert it back to what I had initially in the PR. It seems like we have some logic that depends on us unsubscribing from these streams at the right time, which isn't only when the trigger is destroyed, which got caught by a couple of the unit tests.